### DISCUSSION TITLE
FileVersion broken on Windows

### DISCUSSION THREAD

**Discussion by MWinhard** [rustdesk/rustdesk#7302] [General]
### Bug Description

Version numbers like "1.2.3-1" or "1.2.3+39" are not Windows compatible. Please use numbers like "1.2.3.1" or "1.2.3.39" for file version and product version.

Because of this, version 1.2.3-1 shows an empty detail dialog in Windows Explorer. See screenshot.

### How to Reproduce

Try to read file version of rustdesk-1.2.3-1-x86_64.exe programmatically.

- or -

Right click on rustdesk-1.2.3-1-x86_64.exe in Windows Explorer. Select properties. Show detail page.

### Expected Behavior

File version and product version of rustdesk exe are in Windows compatible format, i.e. up to four numbers, separated by a dot.

### Operating system(s) on local side and remote side

Windows 10

### RustDesk Version(s) on local side and remote side

1.2.3-1

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/111646687/a1daefe5-90e2-4762-b8d0-b634b2b03ac3)

**Comment by MWinhard** [comment]
Why did you move this to discussions?
It's obvious that this is an issue.

**Comment by deep-soft** [comment]
1. That version of installer did not include FileVersion and ProductVersion.
You must go to rustdesk.exe (where it is installed) and check the FileVersion and ProductVersion.

2. Only FileVersion must follow the Windows compatible format, i.e. up to four numbers, the ProductVersion it's not limited to this four numbers format.

See below FileVersion and ProductVersion for nightly version:

a) installer
![image](https://github.com/rustdesk/rustdesk/assets/10506197/d2ad2b84-471e-4f2b-8a5a-30a7a4bff1d1)

b) rustdesk.exe
![image](https://github.com/rustdesk/rustdesk/assets/10506197/b4203953-e2ba-4472-9052-0fa3a588fef5)

**Comment by MWinhard** [comment]
Thank you very much for the quick reply.

Will future File Versions of the installer be Windows compatible, too?
Is there already a planned release date for version 1.2.4?
